zerver/views/streams.py

```diff
@@ -395,10 +391,6 @@ def get_streams_backend(request, user_profile,
                              include_all_active)
     return json_success({"streams": streams})
 
-def get_public_streams_backend(request, user_profile):
-    return get_streams_backend(request, user_profile, include_public=True,
-                               include_subscribed=False, include_all_active=False)
```

Comment:
(From an API design perspective, it's possible the function should have all the options default to False, but I don't think we can safely change the API here.)